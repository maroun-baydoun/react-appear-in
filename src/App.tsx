import React, { Component } from 'react';


class App extends Component {
  render() {
    return (
      <div>React Appear In</div>
    );
  }
}

export default App;
