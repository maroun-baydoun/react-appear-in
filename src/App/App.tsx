import React, { Component } from 'react';
import styled, { createGlobalStyle, ThemeProvider } from 'styled-components';

import Cell from '../Cell';

const GlobalStyle = createGlobalStyle`
  html, body {
    padding: 0;
    margin: 0;
    width: 100%;
    height: 100%;
  }

  body {
    font-family: Arial, Helvetica, sans-serif;
  }
`;

class App extends Component {
  render() {
    return (
        <ThemeProvider theme={{}}>
            <CellContainer>
            <GlobalStyle/>
            {[...Array(100)].map((_, i)  => <Cell key={i}/>)}
          </CellContainer>
        </ThemeProvider>

    );
  }
}

const CellContainer = styled('div')`
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  background-color: grey;
`;

export default App;
