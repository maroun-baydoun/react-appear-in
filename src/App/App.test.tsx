import React from 'react';
import ReactTestRenderer from 'react-test-renderer';

import App from './App';

describe('<App/>', () => {
  it('renders component', () => {
    const renderedComponent = ReactTestRenderer.create(<App/>);
    expect(renderedComponent.toJSON()).toMatchSnapshot();
  });
});
