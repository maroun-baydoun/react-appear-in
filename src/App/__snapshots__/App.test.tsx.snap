// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> renders component 1`] = `
<div
  className="sc-bwzfXH kbjYKv"
>
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
  <div
    className="sc-bdVaJa gYeXqn"
  />
</div>
`;
